### PR TITLE
Deletes old Guardian Egyptian Display fonts

### PR DESCRIPTION
**Pull request by duarte**
This follows #1163 and deletes the fonts that are no longer used in the templates.

In case they do: all instances of _'Guardian Egyptian Web'_ or _'Guardian Egyptian Display'_ (same font) can be replaced with _'Guardian Headline'_ as follows:

egyptiandisplay-thin.otf → GHGuardianHeadline-Light.ttf
egyptiandisplay-light.otf → GHGuardianHeadline-Light.ttf
egyptiandisplay-regular.otf → GHGuardianHeadline-Regular.ttf
egyptiandisplay-medium.otf → GHGuardianHeadline-Medium.ttf
egyptiandisplay-bold.otf → GHGuardianHeadline-Bold.ttf

Note, GHGuardianHeadline-Light.ttf replaces both egyptiandisplay-thin.otf and egyptiandisplay-light.otf.

***

Do not merge until:
* [x] Android — [#6026](https://github.com/guardian/android-news-app/pull/6026) is merged
* [x] iOS — has checked the files don't need to be present

### CODE REVIEW

**Comment by mohammad-haque** [comment]
@duarte 
I see some direct use of these deleted fonts. Although we do have a mapping already in the codebase but they don't cover all the usages. Mappings look like these:

```
return when (typefaceString) {
            // TODO we will remove all egypt-* font as they are deprecated, but keeping it
            // for now for backward compatibility
            "egypt-thin", "egypt-light", "headline-light" -> TypefaceHelper.headlineLight
            "egypt-regular", "headline-regular" -> TypefaceHelper.headlineRegular
            "egypt-medium", "headline-medium" -> TypefaceHelper.headlineMedium
            "headline-semibold" -> TypefaceHelper.headlineSemibold
            "egypt-bold", "headline-bold" -> TypefaceHelper.headlineBold
            "headline-black" -> TypefaceHelper.headlineBlack
            "agate-regular" -> TypefaceHelper.agateRegular
            "agate-bold" -> TypefaceHelper.agateBold
            "display-sans" -> TypefaceHelper.displaySansRegular
            else -> TypefaceHelper.headlineBold
        }
```

I also see some usages of egypt font like these:
```
private fun getAtomsCss(item: ArticleItem): String? {
        return item.atomsCSS?.joinToString(separator = " ", transform = {
            it.replace("serif-text", "'Guardian Text Egyptian Web', Georgia, serif")
                    .replace("serif-headline", "'Guardian Egyptian Web', Georgia, serif")
                    .replace("sans-serif-text", "'Guardian Agate Sans 1 Web', sans-serif")
                    .replace("sans-serif-headline", "'Guardian Text Sans Web', sans-serif")
        })
    }
```
Not sure what `font name` I can replace them with.

**Comment by mohammad-haque** [comment]
EDIT: above mapping for fonts in android is exclusively for `ThrasherTypeface`

**Comment by duarte** [comment]
@mohammad-haque  I think in this bit concerning thrashers:
```
.replace("serif-headline", "'Guardian Egyptian Web', Georgia, serif")
```
where we replacing a `'serif-headline'` font with `"'Guardian Egyptian Web', Georgia, serif"`, it would be ideal to use the font's correct name, `"'Guardian Headline', Georgia, serif"`. It could be worth updating. However, as #1163 aliases the two names, the change shouldn't have any practical effect as far as I can tell!

**Comment by duarte** [comment]
@mohammad-haque should we wait till #6026 is merged on android before merging this?

**Comment by mohammad-haque** [comment]
>  should we wait till #6026 is merged on android before merging this?
That would be better, I think

**Comment by duarte** [comment]
I think we're ok to merge from the Android side, but still waiting on iOS.
@SLambrakis — are you able to have a look and see if it's ok to merge?

**Comment by JamieB-gu** [comment]
@duarte @SLambrakis @mohammad-haque any update on this or can I close?

**Comment by duarte** [comment]
I think the idea of removing our old fonts from the app bundle is still as valid! But we still gotta wait for the check on iOS before merging

**Comment by JamieB-gu** [comment]
> But we still gotta wait for the check on iOS before merging

@aoifemcl15 @faresite @SLambrakis are you able to check on this? If not then can we close this PR (sorry @duarte!)?

**Comment by SLambrakis** [comment]
Discussed with the devs - we are still using these fonts (or something with the same name) in the iOS code, but we don't think that's reliant on the templates repo (we believe we ship the fonts ourselves for the native layer). Go ahead and merge, and we're confident if it does anything really bad we'll spot it in beta.

**Comment by duarte** [comment]
For some reason I can't merge this nor remove the 'do not merge' tag, so will leave that to whoever can merge.

**Comment by SLambrakis** [comment]
@JamieB-gu @webb04 Is one of you OK to approve this from the templates side?

File: ArticleTemplates/assets/scss/base/_typography.scss
Comment (JamieB-gu) [inline ArticleTemplates/assets/scss/base/_typography.scss:170]: Are these weights correct? They're all 100 off the [common name mappings](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#Common_weight_name_mapping):

- Light: 300
- Regular: 400
- Medium: 500
- Bold: 700

cc @paperboyo

**Review by JamieB-gu — COMMENTED** [review]
LGTM, one question from me on the weightings.

File: ArticleTemplates/assets/scss/base/_typography.scss
Comment (paperboyo) [reply ArticleTemplates/assets/scss/base/_typography.scss:170]: Common are correct and used in other places in the Guardian. Looks like here they have just been taken out of a hat 😄.
No idea what making them correct here would mean, though?

Comment (JamieB-gu) [reply ArticleTemplates/assets/scss/base/_typography.scss:170]: @duarte any thoughts?

Comment (duarte) [reply ArticleTemplates/assets/scss/base/_typography.scss:170]: I have nothing to add Re: weight numbers

**Review by JamieB-gu — APPROVED** [review]

File: ArticleTemplates/assets/scss/base/_typography.scss
Comment (JamieB-gu) [reply ArticleTemplates/assets/scss/base/_typography.scss]: Having rebased the changes in this file go away. I think they were already made in #1163 and not being picked up correctly.

**Comment by JamieB-gu** [comment]
> For some reason I can't merge this nor remove the 'do not merge' tag, so will leave that to whoever can merge.

@duarte I think you couldn't merge because the checks hadn't passed. They have now so you should be able to merge it.

**Comment by duarte** [comment]
@JamieB-gu I still do not see a merge option on my end.

**Pull request merged**
